docs/design/persistent-volume-provisioning.md

```diff
@@ -0,0 +1,169 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/docs/design/persistent-volume-provisioning.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# Persistent Volume Provisioning
+
+This document proposes a model for dynamically provisioning [Persistent Volumes](../../docs/user-guide/persistent-volumes.md)
+
+### tl;dr
+
+One new API kind:
+
+A `PersistentVolumeSet` (PVS) is a storage resource provisioned by an administrator.  PVSets do not have a namespace.  Just as a `ReplicationController` maintains a number of replicas of a pod, a `PersistentVolumeSet` maintains a minimum number of replicas of a `PersistentVolume`.  A PVSet creates new volumes from a template up to a maximum replica count.
```

Comment:
Observation, before having read the whole proposal:

PersistentVolumes are like Nodes -- they represent infrastructure, either provisioned by an admin on bare metal or potentially horizontally auto-scaled or provisioned on demand on a public/private cloud. It doesn't seem like there will be a uniform implementation across cloud providers. We don't have NodeSets partly for this reason.

PersistentVolumeClaims are analogous to Pods. We need replication of PVCs, but not independently of the Pods that consume them.  That's discussed in #260.